src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuegenericletter/IssueGenericLetterAboutToStartHandlerTest.java

```diff
@@ -145,4 +148,24 @@ void givenDocumentsAreExist_thenShouldReturnDocumentsList() {
         assertEquals(1, documentSelection.size());
         assertEquals(2, documentSelection.get(0).getValue().getDocumentsList().getListItems().size());
     }
-}
+
+    @Test
+    void givenDocumentEdited_ThenAddToListOptions() {
+        when(callback.getEvent()).thenReturn(ISSUE_GENERIC_LETTER);
+        when(callback.getCaseDetails()).thenReturn(caseDetails);
+        when(caseDetails.getCaseData()).thenReturn(caseData);
```

Comment:
would it not work if you just instantiated the callback with the relevant data, instead of mocking it
` new Callback<>(caseDetails, Optional.of(caseDetails), ISSUE_GENERIC_LETTER, false);`